app/services/publication-service.js

```diff
@@ -190,9 +190,10 @@ export default class PublicationService extends Service {
   }
 
   async getIsViaCouncilOfMinisters(publicationFlow) {
-    const _case = await publicationFlow.case;
-    const subcases = await _case.subcases;
-    return !!subcases.length;
+    const agendaitem = await this.store.queryOne('agendaitem', {
+      'filter[treatments][publication-flows][:id:]': publicationFlow.id,
+    });
+    return agendaitem != null;
```

Comment:
Oddly enough, this doesn't work.
the `queryOne` yields null even though I just linked a publication so it should return something.
`query` does return results

I think we are hitting a cache issue here:
`queryOne` only worked after I rebooted cache and resource.
I'm guessing the path to cache of agendaitem/treatments wasn't invalidated in this process so we keep hitting the stale one.

Reply:
"I'm guessing the path to cache of agendaitem/treatments wasn't invalidated in this process so we keep hitting the stale one"

This is kinda the same issue we have with the yggdrasil government cache?

Reply:
I suggest using the same query as you do in the `linkPublicationFlow`
```
const agendaItemTreatment = await publicationFlow.agendaItemTreatment;
      const agendaitem = await this.store.queryOne('agendaitem', {
        'filter[treatments][:id:]': agendaItemTreatment.id
      });
```

Reply:
"simple one-to-one relation between agendaitem and agenda-item-treatment"
this will not be the case since multiple agenda versions have different agendaitems that are linked to the same treatment, so it will still be a many-to-one. But since there will only ever b 1 treatment and multiple decisions, we wont have to deal with a many-to-many anymore